src/core/control/jobs/CustomExportJob.cpp

```diff
@@ -25,29 +26,28 @@
 
 CustomExportJob::CustomExportJob(Control* control): BaseExportJob(control, _("Custom Export")) {
     // Supported filters
-    filters.insert({_("PDF files"), ExportType(".pdf")});
-    filters.insert({_("PNG graphics"), ExportType(".png")});
-    filters.insert({_("SVG graphics"), ExportType(".svg")});
-    filters.insert({_("Xournal (Compatibility)"), ExportType(".xoj")});
+    filters.insert({_("PDF files"), ExportType(".pdf", "application/pdf")});
+    filters.insert({_("PNG graphics"), ExportType(".png", "application/png")});
+    filters.insert({_("SVG graphics"), ExportType(".svg", "application/svg")});
```

Comment:
This should be:
```suggestion
    filters.insert({_("PNG graphics"), ExportType(".png", "image/png")});
    filters.insert({_("SVG graphics"), ExportType(".svg", "image/svg+xml")});
```

Compare https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types